src/component/marker/MarkPointView.ts

```diff
@@ -143,12 +143,14 @@ class MarkPointView extends MarkerView {
             }
 
             const style = itemModel.getModel('itemStyle').getItemStyle();
+            const z2 = itemModel.getModel('z2').option;
```

Comment:
Usually be 
```js
const z2 = itemModel.get('z2'); 
// or const z2 = itemModel.get('z2', true); 
```
(no need to create a temporary model object, tho there is probably no performance issue here, handy optimization would be better.)